modules/apps/feature-flag/feature-flag-web/src/main/resources/META-INF/resources/js/FeatureFlagList.tsx

```diff
@@ -0,0 +1,122 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayEmptyState from '@clayui/empty-state';
+import ClayList from '@clayui/list';
+import {sub} from 'frontend-js-web';
+import React, {useState} from 'react';
+
+import FeatureFlagToggle from './FeatureFlagToggle';
+
+export type TFeatureFlags = {
+	companyId: number;
+	dependenciesFulfilled: boolean;
+	dependencyKeys: Array<string>;
+	description: string;
+	enabled: boolean;
+	key: string;
+	title: string;
+};
+
+interface IFeatureFlagListProps {
+	featureFlags: Array<TFeatureFlags>;
+}
+
+const FeatureFlagList: React.FC<IFeatureFlagListProps> = ({featureFlags}) => {
+	const [items, setItems] = useState<TFeatureFlags[]>(featureFlags);
+
+	const displayFeatureFlag = (items: TFeatureFlags[]) =>
+		items.map(
+			({
+				companyId,
+				dependenciesFulfilled,
+				dependencyKeys,
+				description,
+				enabled,
+				key,
+				title,
+			}: TFeatureFlags) => {
+				const dependency = dependencyKeys.map((dep: string) => dep);
```

Comment:
I think that I thought the `sub` function only accepted strings, but still doesn't make sense since map returns a new array :confused: 
Anyway... fixed! :relieved: